Change of approach: use deterministic UUIDs for records imported into Nautobot (#28)
nautobot/nautobot-plugin-netbox-importer: README.md

### Importing the data into Nautobot

> As the `nautobot` user account, run `nautobot-server import_netbox_json <json_file> <netbox_version>`, for example `nautobot-server import_netbox_json /tmp/netbox_data.json 2.10.3`.

Reviewer: Could we instead change this to say "From within the Nautobot application environment" so that it's not exactly tightly coupled to production deployments?
```suggestion
From within the Nautobot application environment, run `nautobot-server import_netbox_json <json_file> <netbox_version>`, for example `nautobot-server import_netbox_json /tmp/netbox_data.json 2.10.3`.
```
